README.md

```diff
@@ -326,6 +326,26 @@ Obtain SMTP credentials from a provider for transactional emails. Set the SMTP_U
 
 <hr>
 
+<img src="https://upload.wikimedia.org/wikipedia/commons/4/44/Microsoft_logo.svg" height="50">
+
+- Go to <a href="https://portal.azure.com/" target="_blank">Azure Portal</a>
+- Sign in with your Microsoft account
+- Navigate to **Microsoft Entra ID** (formerly Azure Active Directory) from the left sidebar or search for it
```

Comment:
The instructions are for a new user, so you don't need to sya what they don't see, such as the former name